samples/cli-translator/src/main/resources/application.properties

```diff
@@ -1,5 +1,9 @@
+quarkus.langchain4j.chat-memory.memory-window.max-messages=2
 quarkus.langchain4j.openai.api-key=${OPENAI_API_KEY}
 quarkus.langchain4j.openai.chat-model.temperature=0
 quarkus.langchain4j.openai.timeout=60s
 quarkus.banner.enabled=false
-quarkus.log.level=WARN
\ No newline at end of file
+quarkus.log.level=WARN
+# if you want to log the requests and responses that go to OpenAI:
```

Comment:
Maybe the comment should say that in that case you also need to set `quarkus.log.level` to `INFO`, otherwise they won't really get logged